packages/lab/src/__tests__/__e2e__/list-next/ListNext.cy.tsx

```diff
@@ -8,119 +11,100 @@ const ITEMS: ItemType[] = [
   { label: "list item 3", value: "item 3" },
 ];
 
-const ITEMS_LONG_LABEL: ItemType[] = [
-  {
-    label:
-      "list item 1: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. ",
-    value: "item 1",
-  },
-  {
-    label:
-      "list item 2: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
-    value: "item 2",
-  },
-  {
-    label:
-      "list item 3: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.",
-    value: "item 3",
-  },
-];
-
-// Single select tests
-describe("A single select list", () => {
+// Uncontrolled single select list
+describe("An uncontrolled single select list", () => {
   const SingleSelectList = (props: any) => {
     return (
       <ListNext {...props}>
         {props.children ||
           ITEMS.map((item, index) => {
-            return <ListItemNext key={index}>{item.label}</ListItemNext>;
+            return (
+              <ListItemNext value={item.value} key={index}>
+                {item.label}
+              </ListItemNext>
+            );
           })}
       </ListNext>
     );
   };
 
-  describe("list width", () => {
+  describe("List width", () => {
     it("should render list with given width", () => {
-      cy.mount(SingleSelectList({ style: { width: "80px" } }));
+      cy.mount(<SingleSelectList style={{ width: "80px" }} />);
```

Comment:
@feliciackh we can probably do without this one, given we are passing the width trough styles, we don't need to test it